handlers/friday.go

```diff
@@ -0,0 +1,68 @@
+package handlers
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"time"
+
+	"io"
+	"net/http"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+var (
+	fridayTrigger = "sextou"
+	fridayGifUrl  = "https://media.tenor.com/zGlEbV_bTnIAAAAC/kowalski-familia.gif"
```

Comment:
An enhancement that you could do is to search a gif using the [Tenor API](https://tenor.com/gifapi/documentation#quickstart-search), this is an example request:

```
curl -X GET https://g.tenor.com/v1/search?q=sextou&key=LIVDSRZULELA&limit=8
```

You could make a request and get one of these randomly